### PR TITLE
fix: only output simple error html on unittest

### PR DESCRIPTION
**Pull request by fengmk2**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
否则在单测里面 assert 错误会导致卡死和一大堆无用的字符串

image

**Comment by okoala** [comment]
+1

File: lib/error_view.js [hunk 1]
```diff
@@ -27,21 +27,6 @@ class ErrorView {
   }
 
   /**
-   * get json error
-   *
-   * @return {Object} error object
-   *
-   * @memberOf ErrorView
-   */
-  toJSON() {
```
Comment (fengmk2) [inline lib/error_view.js]: @okoala 这个完全没有地方使用到，直接删除了。

**Comment by dead-horse** [comment]
1.4.6

**Pull request merged**